bindings/python/.evergreen/test.sh

```diff
@@ -5,16 +5,28 @@
 set -o xtrace   # Write all commands first to stderr
 set -o errexit  # Exit the script with error if any of the commands fail
 
-# This library is built by libmongocrypt/.evergreen/compile.sh
+# libmongocrypt is built by libmongocrypt/.evergreen/compile.sh
 evergreen_root="$(cd ../../../; pwd)"
-export PYMONGOCRYPT_LIB=${evergreen_root}/install/libmongocrypt/lib64/libmongocrypt.so
-
-PYTHONS=("/opt/python/2.7/bin/python" \
-         "/opt/python/3.4/bin/python3" \
-         "/opt/python/3.5/bin/python3" \
-         "/opt/python/3.6/bin/python3" \
-#         "/opt/python/pypy/bin/pypy" \  # TODO: PyPy segfaults on RHEL 6.2
-         "/opt/python/pypy3.6/bin/pypy3")
+
+if [ "Windows_NT" = "$OS" ]; then # Magic variable in cygwin
```

Comment:
I see we're testing on windows-64-vs2017-test. Can you use a spawnhost to see if the Python driver "toolchain" (I use that term very loosely here) is available? If not, can you add a windows-64-vs2015-test section so we can test Python 3?

Alternatively, there are one or more Python 3 versions in the server toolchain which is available on all Evergreen distros.

I'm not super worried about expanded testing on macOS. We don't do it for PyMongo either.